lib/Makefile

```diff
@@ -0,0 +1,440 @@
+# CMAKE generated file: DO NOT EDIT!
```

Comment:
This file was also created by cmake in an "in-source" build, can you please remove it?

Reply:
Removed.